import-service/src/handlers/importProductsFile.js

```diff
@@ -0,0 +1,35 @@
+import S3 from 'aws-sdk/clients/s3';
+import { errorResponse, successResponse } from '../utils/responseBuilder';
+import winstonLogger from '../utils/logger';
+import { getSignedUrl } from '../services/s3Service';
+
+const s3 = new S3({ region: 'eu-west-1', signatureVersion: 'v4' });
+const BUCKET = 'rs-school-import-service-bucket';
```

Comment:
You use this constant in several files. Please make the file with shared constants and move this constant there.

Reply:
As I see I use it only here

Reply:
Oh, sorry. This is suitable for serverless.ts only